src/vm/vm.h

```diff
@@ -71,4 +71,7 @@ void move(VirtualMachine *vm, Value val, Dest dest);
 
 void syscall(VirtualMachine *vm);
 
-void dump(VirtualMachine vm);
\ No newline at end of file
+void dump(VirtualMachine *vm);
+
+
+static char* reg_to_string(int num);
```

Comment:
Static functions are essentially private, so there is no use in exposing them through a header file. Could you please remove this?